etl/steps/data/garden/trade/2025-09-22/historic_trade.meta.yml

```diff
@@ -0,0 +1,26 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Trade & Globalization
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 709
```

Comment:
That's an unusual number, I guess the next release is on a known date around 709 days in the future?